dxm/src/test/java/io/pcp/parfait/dxm/Matchers.java

```diff
@@ -0,0 +1,43 @@
+package io.pcp.parfait.dxm;
+
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
+import org.unitils.reflectionassert.ReflectionComparator;
+import org.unitils.reflectionassert.ReflectionComparatorMode;
+import org.unitils.reflectionassert.difference.Difference;
+import org.unitils.reflectionassert.report.impl.DefaultDifferenceReport;
+
+import static org.unitils.reflectionassert.ReflectionComparatorFactory.createRefectionComparator;
+
+public class Matchers {
+
+    static class ReflectiveMatcher extends BaseMatcher<Object> {
```

Comment:
Not as a hamcrest matcher. There are assertions as part of unitils but we need this as a matcher (to assert for items in lists etc..).